apps/client/src/app/pages/portfolio/activities/activities-page.component.ts

```diff
@@ -30,6 +31,7 @@ import { ImportActivitiesDialogParams } from './import-activities-dialog/interfa
 })
 export class ActivitiesPageComponent implements OnDestroy, OnInit {
   public activities: Activity[];
```

Comment:
Can you remove `public activities`?

Reply:
@dtslvr Does it mean to remove the 'activities' variable or modify/remove the access modifiers?

Reply:
Removing the full line (variable), because the activities should go straight to the data source.